src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -67,10 +67,10 @@ static constexpr bool IsReadable(std::wstring_view text)
 
 namespace winrt::Microsoft::Terminal::Control::implementation
 {
-    TermControlAutomationPeer::TermControlAutomationPeer(TermControl* owner,
+    TermControlAutomationPeer::TermControlAutomationPeer(winrt::weak_ref<TermControl> owner,

```

Comment:
I believe it would be better to pass a strong reference here (`winrt::com_ptr`) and only drop it down to a weak pointer down in `_termControl{ owner }`. I think the `weak_ref` constructor is implicit so it should just work.